test/schnorr_ZZZ-fuzz.c

```diff
@@ -33,6 +33,8 @@
 #include <stdio.h>
 
 #define MAX_REPS 10000
+#define MSG_LEN 128
```

Comment:
Should the message and basename lengths be randomized as well?

Reply:
Yes! Good catch.

I've pushed a new version that randomizes those two lengths, as well.

The `msg` and `basename` buffers remain stack-allocated (allocated to a maximum size of `1024`).

The length is randomized by pushing random bytes into the `[msg,basename]_len` integers then mod'ing by the size of the buffer (and adding 1, to avoid 0 length and allow possibly filling the entire buffer).